README.md

```diff
@@ -1 +1,8 @@
-# PythonHW_Iterator_Generator
\ No newline at end of file
+### PythonHW_Iterator_Generator
+
+#### Contains Python 3 code for classes IteratorEnhancedRange and GeneratorEnhancedRange.
+#### Both work similar to standard Python range(int) function, except they are able work with float arguments
```

Comment:
What is the idea behind float support here?

Reply:
In my opinion it is rather pointless to write the exact copy of the existing function. E.g., in some future work when needed I will never use some "range-copy" that I wrote myself - I will use the original `range` instead. You yourself said on one of the seminars that its better to use a library or a function that has already been written. So I decided to enhance my "Range" so that it's able to work with float numbers. And so my "EnhancedRange" is the same as original, except it can do more which, I believe, may be useful for some future coding.